log/log.go

```diff
@@ -184,3 +199,7 @@ func (nl *nilLogger) Debug(args ...interface{}) {
 
 func (nl *nilLogger) Debugf(msg string, args ...interface{}) {
 }
+
+func (nl *nilLogger) Level() Level {
```

Comment:
Since this function is extracted, shouldn't we have some documentation?

Reply:
I followed the convention here to not add a comment, because it seems that we don't have comments on `nilLogger` methods. What do you think?

Reply:
I think we should add documentation for whatever new we introduce. :/

Reply:
Maybe we should add comments also to older exported methods, when encountered ?